guides/security.md

```diff
@@ -30,12 +30,12 @@ If you issue your own assets, you should usually ensure that they can be revoked
 
 Because Stellar’s security is based around public key encryption, it’s critical that an account’s secret seed is not shared. Anyone who has access to the seed effectively has control of the account. However, if someone learns your account’s seed or you accidentally share it with someone who shouldn’t know it, you can remove its ability to control the account with the following steps:
 
-1. Make a new account.
-2. Add the new account as a signer on the compromised account. (Use the [`set options` operation](concepts/list-of-operations.md#set-options).
+1. Make a new key pair.
+2. Add the new public key as a signer on the compromised account. (Use the [`set options` operation](concepts/list-of-operations.md#set-options).
```

Comment:
Oh! I was totally under the impression from an earlier conversation w/ @bartekn that all the signers had to be references to existing accounts, not just keys! That definitely makes this process seem a little less unwieldy.

This also clarifies the confusion I was having here: https://github.com/stellar/docs/pull/149/#discussion_r70694545 — if the signers are just keys, all accounts that accept a compromised key as a signer *do* need to remove it, since there’s no possibility for a transitive signing relationship if the signers are just keys and not references to accounts.